server/src/auth.js

```diff
@@ -485,6 +485,16 @@ async function maySetAuthRoutes(app, basepath = '', _serverconfig = null) {
 		}
 	})
 
+	// helper with some code duplication just to get clientAuthResult from jwt
+	authApi.getClientAuthResult = function (req) {
```

Comment:
I think this should be generalized and also return `forbiddenRoutes`, so that `getForbiddenRoutesForDsEmbedder()` does not have to be called separately inside `q.getSupportedChartTypes()` in `termdb.server.init.js`. 

So, maybe call this function `authApi.getNonsensitiveInfo()`, and it would return `{forbiddenRoutes, clientAuthResults}`, and can be extended easily later.